Dockerfile

```diff
@@ -0,0 +1,17 @@
+# build stage
+FROM node:lts-alpine as build-stage
+WORKDIR '/app'
+RUN apk --no-cache add git
```

Comment:
without git, the npm install crashes as it tries to download some charting libs from git